packages/fluent/scss/checkbox/_variables.scss

```diff
@@ -0,0 +1,234 @@
+    sm: (
+        size: map-get( $kendo-spacing, 4 ),
+        glyph-size: ( map-get( $kendo-spacing, 4 ) - map-get( $kendo-spacing, thin ) ),
+        ripple-size: map-get( $kendo-spacing, 4 ) * 3,
+        indeterminate-size: map-get($kendo-spacing, sm )
+    ),
+    md: (
+        size: map-get( $kendo-spacing, 5 ),
+        glyph-size: ( map-get( $kendo-spacing, 6 ) - map-get( $kendo-spacing, thin ) ),
+        ripple-size: map-get( $kendo-spacing, 5 ) * 3,
+        indeterminate-size: ( map-get( $kendo-spacing, md ) - map-get( $kendo-spacing, thin ))
+    ),
+    lg: (
+        size: map-get( $kendo-spacing, 6 ),
+        glyph-size: ( map-get( $kendo-spacing, 6 ) - map-get( $kendo-spacing, thin ) ),
+        ripple-size: map-get( $kendo-spacing, 6 ) * 3,
+        indeterminate-size: map-get($kendo-spacing, md ) 
+    )
+) !default;
+
+// Checkbox indeterminte sizes
+$kendo-checkbox-indeterminate-sizes: (
+    sm: (
+        size: map-get( $kendo-spacing, sm ),
+        position: ( map-get( $kendo-spacing, 1 ) - map-get( $kendo-spacing, hair )),
+        border-width: map-get( $kendo-spacing, 1 )
+    ),
+    md: (
+        size: ( map-get( $kendo-spacing, md ) - map-get( $kendo-spacing, thin )),
+        position: map-get( $kendo-spacing, 1 ),
+        border-width: ( map-get( $kendo-spacing, 1 ) + map-get( $kendo-spacing, hair ))
+    ),
+    lg: (
+        size: map-get( $kendo-spacing, md ),
+        position: ( map-get( $kendo-spacing, 1) + map-get( $kendo-spacing, hair )),
+        border-width: ( map-get( $kendo-spacing, 1) + map-get( $kendo-spacing, thin ))
+    )
+) !default;
+
+/// Default width of the checbox.
+/// @group checkbox
+$kendo-checkbox-width: map-get( $kendo-spacing, 5 );
+/// Default height of the checbox.
+/// @group checkbox
+$kendo-checkbox-height: map-get( $kendo-spacing, 5 );
+
+/// Default width of indeterminate checkbox.
+/// @group checkbox
+$kendo-checkbox-indeterminate-width: ( map-get( $kendo-spacing, 3) - map-get( $kendo-spacing, thin) );
+/// Default height of indeterminate checkbox.
+/// @group checkbox
+$kendo-checkbox-indeterminate-height: ( map-get( $kendo-spacing, 3) - map-get( $kendo-spacing, thin) );
+
+/// Default top position of indeterminate checkbox.
+/// @group checkbox
+$kendo-checkbox-indeterminate-top: map-get( $kendo-spacing, 1 );
+/// Default left position of indeterminate checkbox.
+/// @group checkbox
+$kendo-checkbox-indeterminate-left: map-get( $kendo-spacing, 1 );
+
+/// Background color of checkbox.
+/// @group checkbox
+$kendo-checkbox-bg: $white !default;
+/// Color of checkbox.
+/// @group checkbox
+$kendo-checkbox-text: initial !default;
+/// Border color of checkbox.
+/// @group checkbox
+$kendo-checkbox-border: $black !default;
+
+/// The outline of a focused checkbox.
+/// @group checkbox
+$kendo-checkbox-focus-outline: get-theme-color(neutral, 130);
+
+/// Background color of hovered checkbox.
+/// @group checkbox
+$kendo-checkbox-hover-bg: $white !default;
```

Comment:
if the color is the same as the base checkbox, better use inherit or kendo-checkbox-bg. The same goes for the text color and border